__tests__/services/whosampled/WhoSampled.service.test.ts

```diff
@@ -221,11 +221,4 @@ describe('WhoSampled.service Test Suite', () => {
       expect(result).toEqual(null);
```

Comment:
While experimenting with expo modules I got a warning saying there's a dependency cycle between the parser util file and the service, went ahead and fixed that.